osc-staging.py

```diff
@@ -53,14 +53,15 @@ def _full_project_name(self, project):
     if project.startswith(('openSUSE', 'SUSE')):
         return project
```

Comment:
So you used ALP? Because if had SUSE:ALP, it would have ended here, no?

Reply:
no, in my concrete usecase I used `devel:Factory:git-workflow` which has been rewritten into `openSUSE:devel:Factory:git-workflow` which does not exist. however the fix for `-p ALP` would now rewrite this into -p `SUSE:ALP`

Reply:
I see. I suppose most users use a fully qualified project name - I was actually suprised seeing this code :)